clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/model/AzureServerGroupDescription.groovy

```diff
@@ -36,12 +36,15 @@ class AzureServerGroupDescription extends AzureResourceOpsDescription {
   String provisioningState
   String application // TODO standardize between this and appName
   String clusterName
-
-  static class AzureImage {
-    String publisher
-    String offer
-    String sku
-    String version
+  String securityGroupName
+  String subnetId
+  List<String> storageAccountNames
+  AzureScaleSetCapacity capacity
+
+  static class AzureScaleSetCapacity {
+    long min
+    long max
+    Boolean useSourceCapacity
   }
```

Comment:
Sure, but since we are not using them please remove them until we add our own autoscaling support as it will likely look quite different.